concurrent.lib.sh

```diff
@@ -505,7 +511,7 @@ concurrent() (
         tail -f * >> "${pipe}" &
         __crt__meta_collector_pids=($!)
 
-        sed -n -u -e '
+        "${__crt__sed}" -n -u -e '
```

Comment:
For consistency, there are a few other instances of `sed` which should be replaced with this variable as well.